ui/components/Expenses/ExpenseItem.tsx

```diff
@@ -18,9 +20,12 @@ export default function ExpenseItem(props: Props): JSX.Element {
   } = useStore();
 
   const isSelected = expense.spendingId === selectedExpenseId;
-  const fundingSchedule = useFundingSchedule(expense.fundingScheduleId);
+  const spendingFunding = useSpendingFunding(expense);
+
   const onClick = () => setCurrentExpense(isSelected ? null : expense.spendingId);
 
+  const { funding, schedule } = spendingFunding.next.length > 0 && spendingFunding.next[0] || { };
```

Comment:
Should really just solve for multiple funding schedules in the UI now.

This will easily break in the future otherwise.

If there were two on the same day, then the funding.nextContributionAmount would be 1/2 what it is supposed to be (as shown by the UI)